src/couchapps/ReqMgr/views/childresubmissionrequests/map.js

```diff
@@ -5,6 +5,6 @@
  */
 function(doc){
     if(doc.InitialTaskPath){
-        emit(doc.InitialTaskPath.split("/")[1], null);
+        emit(doc.InitialTaskPath.split("/")[1], doc.RequestStatus);
```

Comment:
Just for my education. Who is capturing the so 'emitted' event by this java script? And is the set of actions it is about trigger changed somehow when the RequestStatus is included in it, or this is added here just for clarity?

Reply:
This is the code for the so-called CouchDB views, which can contain either a map or a map and reduce function.
It's provided through the CouchDB HTTP APIs, like:
https://cmsweb.cern.ch/couchdb/reqmgr_workload_cache/_design/ReqMgr/_view/childresubmissionrequests?key=%22cmsunified_ACDC1_r-0-Run2017E_DoubleMuon_UL2017_MiniAODv2_210120_130131_8395%22

Seeing the response of this API should make it clear what the change will be. Please let me know otherwise ;)